CMakeLists.txt

```diff
@@ -115,6 +115,8 @@ set(MIOPEN_INSTALL_CXX_HEADERS Off CACHE BOOL "Install MIOpen's C++ header inter
 
 
 # Embedded Build Configuration
+set(MIOPEN_EMBED_DB "" CACHE STRING "Colon separated list of of architecture CU pairs to embed on-disk DBs in the binary. Example gfx906_60:gfx900_56")
```

Comment:
PR #361 says it is semi-colon delimited. Which is it?

Reply:
PING

Reply:
Fixed